tests/unit/falcon_demo_content/DemoContentTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace falcon_demo_content;
+
+class DemoContentTest extends \Codeception\Test\Unit {
+
+  const DEMO_CONTENT_MODULE = 'falcon_demo_content';
+
+  /**
+   * Checks for the presence demo content.
+   * @group basic
+   */
+  public function testDemoContentEnabled()
+  {
+    // Get array of demo content entities uuid and type.
+    $this->assertTrue(
+      \Drupal::moduleHandler()->moduleExists(self::DEMO_CONTENT_MODULE),
+      "Module " . self::DEMO_CONTENT_MODULE . " should be enabled."
+    );
+
+    // Get array of demo content entities uuid and type.
+    $contentEntities = $this->getContentEntities();
+    $entityRepository = \Drupal::service('entity.repository');
+
+    // Checks for the presence entity in database.
+    foreach ($contentEntities as $contentEntity) {
+      $this->assertNotEmpty($entityRepository->loadEntityByUuid($contentEntity['entity_type_id'], $contentEntity['uuid']),
+        "Missing content with uuid = " . $contentEntity['uuid']);
+    }
+  }
+
+  /**
+   * Checks for the missing demo content.
+   * @group basic
```

Comment:
Please, put it into a separate group `demo_content` and test separately at the end of all other tests.

Reply:
Done.